source/opt/module.h

```diff
@@ -86,6 +86,12 @@ class Module {
   std::vector<Instruction*> GetConstants();
   std::vector<const Instruction*> GetConstants() const;
 
+  // Return result id of global value with |opcode|, 0 if not present.
+  uint32_t GetGlobalValue(SpvOp opcode) const;
```

Comment:
At first it was puzzling that this method is not general enough to find *any* kind of global value.
But I see how you only do this to get an OpConstantFalse value.  I guess this is fine.

Similar with AddGlobalValue not being fully general, but we already have the general overload that takes a unique Instruction. So that's fine too.